tests/test.py

```diff
@@ -1378,6 +1225,9 @@ def _fork_acra(self, acra_kwargs, popen_kwargs):
             # we doesn't need in tests waiting closing connections
             'incoming_connection_close_timeout': 0,
             self.ACRA_BYTEA: 'true',
+            'tls_client_id_from_cert': 'true',
```

Comment:
better to remove it from base class to test acra with default parameters as base test case.